src/main/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStep.java

```diff
@@ -0,0 +1,81 @@
+package com.cloudbees.jenkins.plugins.sshagent;
+
+import hudson.Extension;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class SSHAgentStep extends AbstractStepImpl implements Serializable {
+
+    private static final long serialVersionUID = 1641672112447457198L;
```

Comment:
When you are adding a completely new class, there is zero risk of backwards compatibility issues so best practice is to assign `serialVersionUID = 1L`. If you find a class which has shipped without a `serialVersionUID` then you need to generate the previous effective value before modifying the class. The value you have specified here is the effective value of this class if you didn't specify a `serialVersionUID`, but as the class is new you should force it to `1L` instead